config/optional/views.view.apigee_api_catalog.yml

```
@@ -2,18 +2,21 @@ langcode: en
 status: true
 dependencies:
   config:
+    - field.storage.node.body
+    - node.type.apidoc
     - system.menu.main
   module:
     - apigee_api_catalog
+    - node
     - text
+    - user
 id: apigee_api_catalog
 label: 'API Catalog'
 module: views
 description: 'APIs published using the Apigee API Catalog module.'
```

Thinking we can add descriptions for the two views for more clarity:
```suggestion
description: 'Display of the published APIs for the developers.'
```